src/org/usfirst/frc/team2399/robot/OI.java

```diff
@@ -129,5 +135,8 @@ public OI(){
 		climbStopButt.whenPressed(climbStopSpeed);
 		shiftToHotButt.whenPressed(shiftToHot);
 		shiftToDangerousButt.whenPressed(shiftToDangerous);
+		agitatorForwardButt.whenPressed(agitatorForwardSpeed);
+		agitatorBackwardButt.whenPressed(agitatorBackwardSpeed);
+		agitatorStopButt.whenPressed(agitatorStopSpeed);
```

Comment:
Same "Shouldn't the default handle this"? Also, look into the distinction between whenPressed and whileHeld